app/code/Magento/InventoryInStorePickupQuote/Model/Address/SetAddressPickupLocation.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickupQuote\Model\Address;
+
+use Magento\Quote\Api\Data\AddressExtensionInterfaceFactory;
+use Magento\Quote\Api\Data\AddressInterface;
+
+/**
+ * Set Address pickup location
+ * @api
```

Comment:
We don't want to expose any API for `InventoryInStorePickupQuote`.

Reply:
OK